etc/ncbi-files.txt

```diff
@@ -16,11 +16,6 @@
 #    │                       └── GCF_000001405.25_GRCh37.p13_genomic.gff.gz
 #    └── refseq
 #        └── H_sapiens
-#            ├── historical
```

Comment:
With the changes in this PR, NCBI historical RefSeq files do not need to be referenced by this config any longer.